modules/datadog-lambda-forwarder/providers.tf

```diff
@@ -26,3 +26,17 @@ variable "import_role_arn" {
   default     = null
   description = "IAM Role ARN to use when importing a resource"
 }
+
+
+module "datadog_configuration" {
```

Comment:
could we move the datadog provider config to a separate `datadog-provider.tf`? Same as `datadog-private-location-ecs`?